relays/list-relay.conf.json

```diff
@@ -0,0 +1,16 @@
+{
+	"connections": {
+		"local": "ws://localhost:13900"
+	},
+	"bindings": {
+		"table": {
+			"input": [
+				{ "node": "local/default", "pattern": "*" }

```

Comment:
Given that you're only going to process a very specific message (`scores:ranking`), you should use that as a filter here.
Otherwise, you're going to be receiving all messages, and throwing most of them away.